mozilla-mobile/fenix/app/src/main/res/values/strings.xml

```diff
@@ -191,6 +191,8 @@
     <string name="browser_menu_manage_extensions">Manage extensions</string>
     <!-- Browser menu button that opens AMO in a tab -->
     <string name="browser_menu_discover_more_extensions">Discover more extensions</string>
+    <!-- Browser menu description that is shown when extensions are disabled due to extension errors -->
```

Comment:
@gabrielluong Is this string about disabling "extensions" (plural), or just one (singular)? Or can it be both? We would have to call this out in comments because this affects translations in many languages.

Reply:
This could be one or more extensions being disabled due to extension errors.

Reply:
I see, a comment along these lines would work:
"Browser menu description that is shown when one or more extensions are disabled due to extension errors"

Reply:
@gabrielluong Hi, it seems like more strings have landed in the meantime in this PR. Do you think you can update the comment before Dublin by any chance? LMK, thanks!